modules/dag-l1/src/test/scala/org/tessellation/dag/l1/domain/block/BlockRelationsSuite.scala

```diff
@@ -4,23 +4,31 @@ import cats.effect.{IO, Resource}
 
 import org.tessellation.block.generators._
 import org.tessellation.dag.l1.Main
+import org.tessellation.ext.cats.effect.ResourceIO
+import org.tessellation.json.JsonHashSerializer
 import org.tessellation.kryo.KryoSerializer
 import org.tessellation.node.shared.nodeSharedKryoRegistrar
 import org.tessellation.schema.{Block, transaction}
-import org.tessellation.security.Hashed
 import org.tessellation.security.signature.Signed
+import org.tessellation.security.{Hashable, Hashed}
 
 import weaver.MutableIOSuite
 import weaver.scalacheck.Checkers
 
 object BlockRelationsSuite extends MutableIOSuite with Checkers {
 
-  type Res = KryoSerializer[IO]
+  type Res = Hashable[IO]
 
-  override def sharedResource: Resource[IO, BlockServiceSuite.Res] =
-    KryoSerializer.forAsync[IO](Main.kryoRegistrar ++ nodeSharedKryoRegistrar)
+  def sharedResource: Resource[IO, Res] = for {
+    implicit0(ks: KryoSerializer[IO]) <- KryoSerializer.forAsync[IO](Main.kryoRegistrar ++ nodeSharedKryoRegistrar)
+    implicit0(j: JsonHashSerializer[IO]) <- JsonHashSerializer.forSync[IO].asResource
```

Comment:
Is that notation so that the `KryoSerializer` and `JsonHashSerializer` are implicitly available for `Hashable`?

Reply:
yes. we have a better-monadic-for in the dependencies, so we can use it here: https://github.com/oleg-py/better-monadic-for?tab=readme-ov-file#define-implicits-in-for-comprehensions-or-matches

That's way it's easier to read and extend than normal chaining using `.map` and `.flatMap`